convex/customFunctionExample.ts

```diff
@@ -13,6 +13,7 @@ import { getUserByTokenIdentifier } from "./lib/withUser";
 import { Rules } from "convex-helpers/server/rowLevelSecurity";
 import { DataModel, Doc } from "./_generated/dataModel";
 import { SessionIdArg, vSessionId } from "convex-helpers/server/sessions";
+import { ReaderWithFilter } from "convex-helpers/server/filterWith.js";
```

Comment:
Why does this need the .js to work?

Reply:
You need to add an entrypoint to package.json for the non-node-routing to find it